package.json

```diff
@@ -60,7 +60,7 @@
     "scripts/reanimated_utils.rb",
     "mock.js",
     "plugin/index.js",
-    "plugin/build/",
+    "plugin/build/plugin.js",
```

Comment:
I know it's a simple change but maybe it would be better to have it as a separate PR?

Reply:
We can.